config/tech-docs.yml

```
@@ -30,5 +31,9 @@ max_toc_heading_level: 3
 prevent_indexing: true
 
 show_contribution_banner: true
-github_repo: ministryofjustice/template-documentation-site
+github_repo: ministryofjustice/data-catalogue-runbooks
 github_branch: main
+
```

```suggestion

default_owner_slack: "#data-catalogue"
owner_slack_workspace: moj.enterprise
```

Haven't tested this - but there is a way to indicate slack links in the review banner